tests/end2end-tests/graphstorm-nc/mgpu_test.sh

```diff
@@ -249,6 +249,32 @@ error_and_exit $?
 
 rm /tmp/train_log.txt
 
+echo "**************dataset: MovieLens classification, RGCN layer: 1, node feat: BERT nodes: movie, user inference: mini-batch save model save emb node, use wholegraph for cache_lm_embed"
+python3 -m graphstorm.run.gs_node_classification --workspace $GS_HOME/training_scripts/gsgnn_np/ --num-trainers $NUM_TRAINERS --num-servers 1 --num-samplers 0 --part-config /data/movielen_100k_lm_encoder_train_val_1p_4t/movie-lens-100k-text.json --ip-config ip_list.txt --ssh-port 2222 --cf ml_nc_utext.yaml  --save-model-path /data/gsgnn_nc_ml_text/ --topk-model-to-save 1 --save-embed-path /data/gsgnn_nc_ml_text/emb/ --num-epochs 3 --logging-file /tmp/train_log.txt --cache-lm-embed true --preserve-input True --use-wholegraph-embed true
+
+error_and_exit $?
+
+cnt=$(ls -l /data/gsgnn_nc_ml_text/ | grep epoch | wc -l)
+if test $cnt != 1
+then
+    echo "The number of save models $cnt is not equal to the specified topk 1"
+    exit -1
+fi
+
+best_epoch=$(grep "successfully save the model to" /tmp/train_log.txt | tail -1 | tr -d '\n' | tail -c 1)
```

Comment:
Can you also check the whether the cached Bert embeddings are saved under /data/movielen_100k_lm_encoder_train_val_1p_4t/? And can you check the meta.json?

Reply:
I mean whether the `wg-embed_part_0_of_2`,  `wg-embed_part_1_of_2`, ..., are under `/data/movielen_100k_lm_encoder_train_val_1p_4t/`